generation_config.yaml

```diff
@@ -0,0 +1,60 @@
+gapic_generator_version: 2.40.1-SNAPSHOT
+protoc_version: '25.3'
+googleapis_commitish: 5a0ae652df1ef39b7849bf2bd6ba417376ce3204
+libraries_bom_version: 26.38.0
```

Comment:
I don't think this is needed for this repo.

Reply:
When I added this parameter, I didn't think it may not be needed in `README.md`.

I'll change it to optional.

Reply:
On a second thought, `libraries_bom_version` is used in google-cloud-java and handwritten libraries. Making it an optional parameter may cause the README.md rendered incorrectly.

Should we just use an empty string to indicate this value is actually not used in this repo?

Reply:
Let's change it to optional.

Reply:
Done.